### PR TITLE
Update to latest Fuel versions

### PR DESCRIPTION
**Pull request by Dentosal**

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -46,91 +46,78 @@ forc parse-bytecode out/debug/dbg_example.bin
...
 ```
 
-We can recognize the `while` loop by the conditional jumps `JNZI(..)`. The condition just before the first jump can be identified by `LT(..)` instruction (for `<`). Some notable instructions that are generated only once in our code include `MUL(..)` for multiplication and `LOG(.., 0, 0, 0)` from the `log` function.
+We can recognize the `while` loop by the conditional jumps `JNZI`. The condition just before the first jump can be identified by `LT` instruction (for `<`). Some notable instructions that are generated only once in our code include `MUL` for multiplication and `LOG {.., 0, 0, 0}` from the `log` function.
 
 ## Setting up the debugging
 
-We can start up the debug infrastructure. On a new terminal session run `fuel-core --db-type in-memory`; we need to have that running because it actually executes the program. Now we can fire up the debugger itself: `fuel-debugger`. Now
+We can start up the debug infrastructure. On a new terminal session run `fuel-core --db-type in-memory --debug`; we need to have that running because it actually executes the program. Now we can fire up the debugger itself: `fuel-debugger`. Now
```
Comment (sdankel) [inline docs/walkthrough.md]: Is this the same as `fuel-core run --db-type in-memory --debug` ?

File: docs/walkthrough.md [hunk 2]
```diff
@@ -46,91 +46,78 @@ forc parse-bytecode out/debug/dbg_example.bin
...
 if everything is set up correctly, you shoould see the debugger prompt (`>> `). You can use `help` command to list available commands.
 
 Now we would like to inspect the program while it's running. To do this, we first need to send the script to the executor, i.e. `fuel-core`. To do so, we need a *transaction specification*, `tx.json`. It looks something like this:
 
 ```json
```
Comment (sdankel) [inline docs/walkthrough.md:81]: Is this file generated somehow, or are users meant to create it by hand?

File: docs/walkthrough.md [hunk 3]
```diff
@@ -142,24 +129,7 @@ However, the key `script` in should contain the actual bytecode to execute, i.e.
 python3 -c 'print(list(open("out/debug/dbg_example.bin", "rb").read()))'
```
Comment (sdankel) [inline docs/walkthrough.md:129]: On line 126, I think it should be "However, the key is `script` should contain"

File: docs/walkthrough.md (hunk 1 above)
Comment (Dentosal) [reply docs/walkthrough.md]: No but it should be. Good catch.

File: docs/walkthrough.md (hunk 2 above)
Comment (Dentosal) [reply docs/walkthrough.md:81]: The JSON file is created by hand. Maybe I should write the instructions to do that somewhere, but currently the process is quite cumbersome. I'm hoping to build a tool soon.

File: docs/walkthrough.md (hunk 3 above)
Comment (Dentosal) [reply docs/walkthrough.md:129]: Good catch

**Review by sdankel — APPROVED** [review]

**Pull request merged**